### PR TITLE
Fix/Fixed the space in the homepage right items issues #125 

### PR DESCRIPTION
**Pull request by Emsun0211**

### CODE REVIEW

**Comment by JulianaSau** [comment]
Hi @Emsun0211 Please close the other pr you opened. Also, indicate the linear ticket number that you're working on in the description of your pr

File: yarn.lock [hunk 1]
```diff
@@ -3345,7 +3345,7 @@
 
 "@jridgewell/gen-mapping@^0.3.0":
   version "0.3.2"
-  resolved "https://registry.yarnpkg.com/@jridgewell/gen-mapping/-/gen-mapping-0.3.2.tgz#c1aedc61e853f2bb9f5dfe6d4442d3b565b253b9"
```
Comment (JulianaSau) [inline yarn.lock:3348]: Ensure that you do not modify anything from this file. You can discard these changes from your pr as they will interfere with other collaborators on the repo.
when you're done, you can tag me to confirm that you've made the changes

**Review by JulianaSau — CHANGES_REQUESTED** [review]

File: yarn.lock (hunk 1 above)
Comment (Emsun0211) [reply yarn.lock:3348]: Ok. Thanks. Hope I haven't made changes to the yarn.lock already?

Comment (JulianaSau) [reply yarn.lock:3348]: No you haven't. The changes only reflect if your or is merged

**Pull request closed without merging**